src/compiler/execution.jl

```diff
@@ -238,15 +245,29 @@ function Base.show(io::IO, ::MIME"text/plain", k::AbstractKernel{F,TT}) where {F
     print(io, "CUDA.$(nameof(typeof(k))) for $(k.f)($(join(TT.parameters, ", ")))")
 end
 
-@inline @generated function call(kernel::AbstractKernel{F,TT}, args...; call_kwargs...) where {F,TT}
+@inline @generated function (kernel::AbstractKernel{F,TT})(args...; call_kwargs...) where {F,TT}
```

Comment:
can this be made vararg like it was previously? x/ref https://github.com/EnzymeAD/Enzyme.jl/issues/1745

Not sure if thats the fix, but it feels like it

Reply:
The call is still vararg.

Can you adapt the extension to handle this version of kernel calls?

Reply:
sorry by vararg I mean `args::Vararg{Any,N}`

Reply:
Same error.